Classes/GlobalStateExplorers/FLEXInstancesTableViewController.m

```diff
@@ -103,20 +181,27 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
         cell.detailTextLabel.font = cellFont;
         cell.detailTextLabel.textColor = [UIColor grayColor];
     }
-    
-    id instance = self.instances[indexPath.row];
-    NSString *title = nil;
-    if ((NSInteger)[self.fieldNames count] > indexPath.row) {
-        title = [NSString stringWithFormat:@"%@ %@", NSStringFromClass(object_getClass(instance)), self.fieldNames[indexPath.row]];
-    } else {
-        title = [NSString stringWithFormat:@"%@ %p", NSStringFromClass(object_getClass(instance)), instance];
-    }
+
+    NSArray *row = self.sections[indexPath.section][indexPath.row];
+    NSString *title = row[0];
+    id instance = row[1];
+
     cell.textLabel.text = title;
     cell.detailTextLabel.text = [FLEXRuntimeUtility descriptionForIvarOrPropertyValue:instance];
     
     return cell;
 }
 
+- (NSString *)tableView:(UITableView *)tableView titleForHeaderInSection:(NSInteger)section {
+    switch (section) {
+        case 0: return nil;
```

Comment:
Would be nice if this was a bit more driven by data we construct in init. Could have a section object with title + rows fields.

Reply:
I think my recent changes about wrap this up! Tested and working